src/app/view/_routes/editorganization/editorganization.component.html

```diff
@@ -46,16 +46,45 @@
         <button mat-button matStepperNext type="button">Next</button>
       </div>
     </mat-step>
-    <mat-step formGroupName="2" [stepControl]="formArray?.get([2])" [optional]="true">
+    <mat-step formGroupName="2" [optional]="true">
       <ng-template matStepLabel>Admin management</ng-template>
-      <mat-form-field>
-        <mat-label>Manager</mat-label>
-        <input matInput formControlName="managerEmailCtrl" placeholder="Ex. mariorossi@gmail.com" />
+      <mat-list>
+        <div *ngFor="let admin of administrators">
+          <mat-list-item class="fieldName">{{ admin.role.viewValue }}</mat-list-item>
+          <mat-list-item class="fieldData">{{ admin.email }}</mat-list-item>
+          <button
+            mat-button
+            matTooltip="Remove from administrators"
+            aria-label="Button that delete this administrator when pressed"
```

Comment:
deleteS <-

ma io lo cambierei in
"remove this administrator from the organization"

Reply:
è un aria-label, semplicemente un testo alternativo che appare se l'icona non dovesse caricarsi, cosa impossibile praticamente visto che le icone sono un font di material

Reply:
si so a cosa serve l'aria-label, stavo facendo notare che "button that delete" è scorretto, dovrebbe essere "button that deletes"